crates/router/src/connector/stripe/transformers.rs

```diff
@@ -3674,3 +3677,601 @@ mod test_validate_shipping_address_against_payment_method {
+            individual_dob_year: individual_details.individual_dob_year,
+            individual_address_line1: address.line1,
+            individual_address_line2: address.line2,
+            individual_address_postal_code: address.zip,
+            individual_address_city: address.city,
+            individual_address_state: address.state,
+            individual_email: Some(customer_email),
+            individual_phone: customer_details.phone,
+            individual_id_number: individual_details.individual_id_number,
+            individual_ssn_last_4: individual_details.individual_ssn_last_4,
+        })
+    }
+}
+
+// Recipient creation response transform
+#[cfg(feature = "payouts")]
+impl<F> TryFrom<types::PayoutsResponseRouterData<F, StripeConnectRecipientCreateResponse>>
+    for types::PayoutsRouterData<F>
+{
+    type Error = Error;
+    fn try_from(
+        item: types::PayoutsResponseRouterData<F, StripeConnectRecipientCreateResponse>,
+    ) -> Result<Self, Self::Error> {
+        let response: StripeConnectRecipientCreateResponse = item.response;
+
+        Ok(Self {
+            response: Ok(types::PayoutsResponseData {
+                status: Some(enums::PayoutStatus::RequiresPayoutMethodData),
+                connector_payout_id: response.id,
+                payout_eligible: None,
+            }),
+            ..item.data
+        })
+    }
+}
+
+// Recipient account's creation request
+#[cfg(feature = "payouts")]
+impl<F> TryFrom<&types::PayoutsRouterData<F>> for StripeConnectRecipientAccountCreateRequest {
+    type Error = Error;
+    fn try_from(item: &types::PayoutsRouterData<F>) -> Result<Self, Self::Error> {
+        let request = item.request.to_owned();
+        let payout_method_data = item.get_payout_method_data()?;
+        let customer_details = request
+            .customer_details
+            .get_required_value("customer_details")
+            .change_context(errors::ConnectorError::MissingRequiredField {
+                field_name: "customer_details",
+            })?;
+        let customer_name = customer_details
+            .name
+            .get_required_value("name")
+            .change_context(errors::ConnectorError::MissingRequiredField {
+                field_name: "customer_details.name",
+            })?;
+        let payout_vendor_details = request
+            .vendor_details
+            .get_required_value("vendor_details")
+            .change_context(errors::ConnectorError::MissingRequiredField {
+                field_name: "vendor_details",
+            })?;
+        let (_vendor_details, individual_details) = (
+            payout_vendor_details.vendor_details,
+            payout_vendor_details.individual_details,
+        );
+        match payout_method_data {
+            api_models::payouts::PayoutMethodData::Card(_c) => {
+                Ok(Self::Token(RecipientTokenRequest {
+                    external_account: "tok_visa_debit".to_string(),
```

Comment:
assign this to constant and use

Reply:
We need to add another API call here, which would return a tokenised card. For now, can we let it be as it is?